lint.go

```diff
@@ -1159,11 +1159,17 @@ func (f *file) lintErrors() {
 	}
 }
 
-func lintCapAndPunct(s string) (isCap, isPunct bool) {
+func lintCapAndPunct(s string) (isCap, isPunct, isNewline bool) {
 	first, firstN := utf8.DecodeRuneInString(s)
 	last, _ := utf8.DecodeLastRuneInString(s)
 	isPunct = last == '.' || last == ':' || last == '!'
+	isNewline = last == '\n'
 	isCap = unicode.IsUpper(first)
+	if isNewline && strings.Index(s, "\n") != len(s)-1 {
+		// Don't flag strings that contain more than one newline. They
```

Comment:
**Number of packages (includes forks and vendored code):** 251171

**Lines of code matching, including vendored code and forks:**
**without the exception:** 8873 (1 in cmd/link/internal/ld)
**with the exception:** 8240 (1 in cmd/link/internal/ld)

**Repositories/projects on github, bitbucket and gopkg.in, including forks:**
**without the exception:** 1975
**with the exception:** 1841

Looking at some of the matches filtered by the exception, it is questionable whether the exception should exist or not. They're errors of this kind:

`errors in package %q:\n%v\n`

or

`unknown command %q\nRun 'help' for usage.\n`

I'm not sure if multi-line errors should end with or without a final newline.